media/css/mozorg/home/home.scss

```diff
@@ -2,780 +2,94 @@
-        background-size: auto;
-        padding: 60px 0;
-
-        .intro-content {
-            width: calc(50% - 40px);
-        }
-
-        .intro-image {
-            padding-right: 40px;
-            width: calc(50% - 40px);
-
-            img {
-                margin: 0 auto;
-            }
-        }
-    }
-
-    @media #{$mq-desktop-wide} {
-        padding: 80px 0;
-    }
+.extensions .mzp-c-card-picto-content::before {
+    @include at2x('/media/img/home/externals-sprite.png', 100px, 300px);
 }
 
-html[dir="rtl"] {
-    #mozilla {
-        background-position: right 20px top;
-
-        .intro-image {
-            transform: scaleX(-1);
-        }
-
-        @media #{$mq-tablet} {
-            .intro-content {
-                padding-left: 0;
-                padding-right: 40px;
-                text-align: right;
-            }
-        }
-    }
-
-    #quantum {
-        @media #{$mq-desktop} {
-            text-align: right;
-
-            #fx-logo:before {
-                margin-left: 10px;
-                margin-right: 0;
-            }
-        }
-    }
-}
-
-#mozilla {
-    @media #{$mq-tablet} {
-        .content {
-            @include clearfix;
-        }
-
-        .intro-content {
-            float: right;
-        }
-    }
+.careers .mzp-c-card-picto-content::before {
+    @include at2x('/media/img/home/externals-sprite.png', 100px, 300px);
+    background-position: left -100px;
 }
 
-// *** IMPACT/INNOVATE *** //
-.section-content {
-    @include border-box;
-    min-width: $width-phone;
-    padding: 10px 20px 20px;
-
-    .section-headline {
-        @include font-size-level3;
-        margin-bottom: 20px;
-    }
-
-    p {
-        padding-bottom: 20px;
-    }
-
-    a {
-        @include open-sans();
-    }
-
-    @media #{$mq-tablet} {
-        padding: 10px 40px 20px;
-    }
-
-    @media #{$mq-desktop} {
-        // line up with left edge of mozilla logo in nav
-        &:first-child {
-            padding-left: $nav-margin;
-        }
-
-        // equal padding on right side
-        &:last-child {
-            padding-right: $nav-margin;
-        }
-    }
+.help .mzp-c-card-picto-content::before {
+    @include at2x('/media/img/home/externals-sprite.png', 100px, 300px);
+    background-position: left -200px;
 }
 
-#impact-innovate-wrapper {
-    margin: 0 auto;
-    padding: 40px 0 20px;
-
-    @media #{$mq-tablet} {
-        padding: 80px 0;
-    }
-
-    @media #{$mq-desktop-wide} {
-        max-width: $width-max-content;
-    }
-}
+//* -------------------------------------------------------------------------- */
+// Two column section
 
-// two column faux table layout for wider viewports
-@media #{$mq-tablet} {
-    #impact-innovate-wrapper {
-        @include clearfix;
-        padding-bottom: 80px;
-    }
+.c-column {
+    box-sizing: border-box;
+    display: block;
+    padding: $spacing-xl 0;
 
-    #impact,
-    #innovate {
+    @media #{$mq-lg} {
         float: left;
```

Comment:
This section needs a little work for RTL languages:

![image](https://user-images.githubusercontent.com/400117/72058201-96e9b200-32c7-11ea-98ba-1c5ce21d6f71.png)